src/App.tsx

```diff
@@ -1,57 +1,29 @@
-import React from 'react';
-import './App.scss';
+import React, { useState } from 'react';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import './App.scss';
+import { DropList } from './components/DropList';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPeople, setSelectedPeople] = useState<Person | null>(null);
```

Comment:
it is a person, right? not people
```suggestion
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
```

Reply:
also it will be good to have a button to reset selected person